src/slots/Slot.js

```diff
@@ -20,13 +21,15 @@ class Slot {
 
   static get UNIT_TYPE_UNIT_OPTIONS() {
     return {
-      [this.UNIT_TYPES.TEMPERATURE]: ['K', '°C', '°F'],
-      [this.UNIT_TYPES.ROTATIONAL_SPEED]: ['rpm'],
-      [this.UNIT_TYPES.PERCENTAGE]: ['%'],
-      [this.UNIT_TYPES.UNITLESS]: ['-'],
+      [Slot.UNIT_TYPES.TEMPERATURE]: ['K', '°C', '°F'],
+      [Slot.UNIT_TYPES.ROTATIONAL_SPEED]: ['rpm'],
+      [Slot.UNIT_TYPES.PERCENTAGE]: ['%'],
+      [Slot.UNIT_TYPES.UNITLESS]: ['-'],
     }
   }
 
+  static get UNITLESS_UNIT() { return Slot.UNIT_TYPE_UNIT_OPTIONS.unitless[0] }
```

Comment:
Here you are implicitly using the "magic string" `unitless`. A safer options would be 
```js
Slot.UNIT_TYPE_UNIT_OPTIONS[Slot.UNIT_TYPES.UNITLESS][0]
```

Reply:
v good 👁️ ! updated